database/insertCsv.js

```diff
@@ -0,0 +1,26 @@
+const connection = require('./index')
+
+insertCsv = () => { 
+  const queryStr = `LOAD DATA LOCAL INFILE '/Users/jbishow/Documents/Javascripts/Hack Reactor/System Design Capstone/component-cp/database/csv/data.csv' INTO TABLE chairs FIELDS TERMINATED BY ',' LINES TERMINATED BY '\n' IGNORE 1 ROWS;`
+  return connection.query(queryStr)
+}
+
+indexTable = () => {
+  const queryStr = `ALTER TABLE chairs ADD INDEX collection_id (collection_id);`
+  return connection.query(queryStr)
+}
+
+insertCsv()
+  .then(()=>console.log('inserted csv'))
+  .then(()=> {
+    indexTable()
+      .then(() => {
+        console.log('indexed table')
+      })
+      .catch((err) => {
+        console.log('error indexing mysql table')
+      })
+  })
+  .catch((err) => {
+    console.log('error inserting csv')
+  })
```

Comment:
I like that you have this as a script you can run as opposed to having to do it in the terminal, very cool :)